### DISCUSSION TITLE
Failed to download VS Code Server

### DISCUSSION THREAD

**Discussion by in-in** [VSCodium/vscodium#1422] [Q&A]
I want to use Dev Containers, but when I connect to a container I get an error
![23053183154](https://user-images.githubusercontent.com/8797432/220696645-1b9d4bf8-70ca-40d6-95e9-c471728dead1.png)

How can I fix it?

In VS Code Dev Containers works without any problems

codium: 1.75.1

**Comment by Grievous-Spinout** [comment]
same problem here

**Comment by pentamassiv** [comment]
There is an issue for this: https://github.com/VSCodium/vscodium/issues/1229
